video_store.py

```diff
@@ -0,0 +1,82 @@
+import requests
+import datetime
+
+class VideoStore:
+    def __init__(self, url = "http://localhost:5000"):
```

Comment:
Great job separating the request side and the menu side of the project.  I notice that there are a lot of similarities between the video and customer functions.  I recommend finding ways to combine the functionality into a helper function, for example a function like:
```
    def delete(self, path, id):
        response = requests.delete(self.url+f"/path/{id}")
        return response.json()
```
could be called by both delete_video and delete_customer.